packages/@mantine/charts/src/ChartTooltip/ChartTooltip.tsx

```diff
@@ -1,3 +1,4 @@
+import _ from 'lodash';
```

Comment:
`@mantine/charts` package does not depend on `lodash`, this dependency cannot be used.